packages/babel-preset-udemy-website/README.md

```diff
@@ -6,9 +6,10 @@ This is the main configuration used by our
 
 # Installation
 
-Install [Babel](https://www.github.com/babel/babel) locally.
+Install [Babel](https://www.github.com/babel/babel) locally. See this package's package.json
```

Comment:
@babel/core used to be a peer dep. Then it was a direct dep. Now, it's a peer dep again. website-django should load @babel/core.